worlds/sc2/client.py

```diff
@@ -271,7 +273,7 @@ def print_faction_title():
                         print_faction_title()
                         has_printed_faction_title = True
                         (ColouredMessage('* ').item(item.item, self.ctx.slot, flags=item.flags)
-                            (" from ").location(item.location, self.ctx.slot)
+                            (" from ").location(item.location, item.player)
```

Comment:
I remember making a change like this because it was choking on something. Is this robust to displaying items sent from the server (ie from `/send` and `start_inventory`?)

Reply:
`start_inventory` shows up as `Server`, `/send` and `!getitem` show up as `Cheat Console`, `/send_location` shows up as the correct location. The latter is how I found this bug in the first place.